camayoc/tests/qpc/api/v1/conftest.py

```diff
@@ -259,3 +262,26 @@ def run_all_scans(vcenter_client, session_cleanup):
         for vm in vcenter_vms:
             if vm.runtime.powerState == 'poweredOn':
                 vm.PowerOffVM_Task()
+
+
+def scan_list():
```

Comment:
We have similar code on CLI's conftest and it seems to be something required for all the user interfaces we have. We can consider having these helpers to utils, what do you think?

See https://github.com/quipucords/camayoc/blob/master/camayoc/tests/qpc/cli/test_scans.py#L38-L51

Reply:
Yeah I think that utilties to list sources, credentials, scans, and inventory items would be good things to have in a top level config. I'll write a separate "enhancement" issue for that.